### PR TITLE
JENKINS-64088 Fix plugin on 2.264

### PR DESCRIPTION
**Pull request by timja**
I've added incrementals and bom to make testing a pre-release simple

2.204.x is the current conservative baseline, although official docs are recommending slightly newer (https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/)

The code changes are to make spotbugs pass on newer versions of it

### CODE REVIEW

**Review by fqueiruga — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Post build task
+
+This plugin allows the user to execute a shell/batch task depending on
+the build log output. Java regular expression are allowed.
+
+This feature allows user to associate shell or a batch scripts that
+perform some tasks on Hudson depending on the build log output. If the
```
Comment (oleg-nenashev) [inline README.md:7]:
```suggestion
perform some tasks on Jenkins depending on the build log output. If the
```

File: pom.xml [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
-	<url>http://wiki.hudson-ci.org/display/HUDSON/Post+build+task</url>
+	<properties>
+		<revision>1.9</revision>
+		<changelist>-SNAPSHOT</changelist>
+		<gitHubRepo>jenkinsci/postbuild-task-plugin</gitHubRepo>
+		<jenkins.version>2.204.6</jenkins.version>
+		<java.level>8</java.level>
+	</properties>
+	
+	<description>Allows the user to execute a shell/batch task depending on the build log output. Java regular expression are allowed. If the string matches anywhere in the build log output, the script will execute.</description>
```
Comment (oleg-nenashev) [inline pom.xml:23]: Might be a bit long for UI

**Review by oleg-nenashev — APPROVED** [review]
Looks about right.

File: pom.xml (hunk 2 above)
Comment (timja) [reply pom.xml:23]: this doesn't get shown in the UI, index.jelly does https://github.com/jenkinsci/postbuild-task-plugin/blob/5d49bd8cfcbac24134fa09fbfeeda70ed1604e70/src/main/resources/index.jelly#L28

**Review by oleg-nenashev — APPROVED** [review]
GitHub permissions should be granted now @timja

**Comment by OxingchenO** [comment]
This is useful to upgrade from 1.8 to 1.9

**Pull request merged**